src/assets/UserSummary.css

```diff
@@ -0,0 +1,55 @@
+.user-profile-card {
+  width: 250px !important;
```

Comment:
Do we really need all these `!important`?

Reply:
maybe there are a few that we don't need, but in general yes, if we want to use a CSS framework like semantic ui